ajax/libs/daterangepicker/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "daterangepicker",
+  "filename": "daterangepicker.min.js",
+  "version": "1.3.22",
+  "description": "Date range picker component for Bootstrap",
+  "main": "daterangepicker.js",
+  "style": "daterangepicker-bs3.css",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/dangrossman/bootstrap-daterangepicker.git"
+  },
+  "author": {
+    "name": "Dan Grossman",
+    "email": "dan@dangrossman.info",
+    "url": "http://www.dangrossman.info"
+  },
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/dangrossman/bootstrap-daterangepicker/issues"
+  },
+  "homepage": "https://github.com/dangrossman/bootstrap-daterangepicker",
+  "engines": {
+    "node": "0.10.32",
+    "nmp": "1.4.28"
+  },
+  "dependencies": {
+    "jquery": "^1.10",
+    "moment": "^2.9.0"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com:ghiden/angucomplete-alt.git",
```

Comment:
And what's the reason why that you added a totally unrelated repo as its source here?